deploy/k8s/DEPLOYING.md

```diff
@@ -1,43 +1,48 @@
 # Deploying
 
-This is more a collection of notes rather than a proper documentation. Besides, neither CRC nor Minikube work at this
-moment, for various reasons.
+## With Minikube
 
-### With CRC
-
-Get and setup `crc`, then start `crc`:
+Start `minikube`:
 
 ```shell
-crc start --cpus 8 --memory 32768 --disk-size 80
+minikube start --cpus 8 --memory 24576 --disk-size 20gb --addons ingress
 ```
 
-Next, deploy the application:
+Create a namespace:
 
 ```shell
-
-cd deploy/k8s
-helm dependency update chart/
-helm upgrade --install --debug -n trustification --create-namespace trustification chart/ --values values-crc.yaml
+kubectl create ns trustification
 ```
 
-### With Minikube
+Then, deploy the application:
 
-**NOTE:** With Minikube there's currently no ingress. Which might make is pretty useless.
+```shell
+helm upgrade --install -n trustification infrastructure charts/trustification-infrastructure --values values-minikube.yaml --set-string keycloak.ingress.hostname=sso.$(minikube ip).nip.io --set-string appDomain=.$(minikube ip).nip.io
+helm upgrade --install -n trustification trustification charts/trustification --values values-minikube.yaml --set-string appDomain=.$(minikube ip).nip.io
+```
 
-Start `minikube`:
+Speed up running initial jobs:
 
 ```shell
-minikube start --cpus 8 --memory 24576 --disk-size 20gb --addons ingress
+kubectl -n trustification create job --from=cronjob/v11y-walker v11y-walker-initial
 ```
 
```

Comment:
Suggestion:
To access to the ui, use `minikube dashboard` to open it in the browser , then in the ingress section search the row of the spog-ui and copy the value in the hosts column, use that link with http instead of https. f asked username and password could be founded in the values-minikube.yaml

Reply:
I added a few words on that too.